lib/govspeak.rb

```diff
@@ -269,11 +269,18 @@ def render_image(image)
       lines = []
       lines << %{<figure#{id_attr} class="image embedded">}
       lines << %{<div class="img"><img src="#{encode(image.url)}" alt="#{encode(image.alt_text)}"></div>}
-      lines << %{<figcaption>#{encode(image.caption)}</figcaption>} if image.caption
+      lines << '<figcaption>' if figcaption?(image)
+      lines << %{<p>#{encode(image.caption)}</p>} if image.caption
+      lines << %{<p>#{encode(t('govspeak.image.figure.credit', credit: image.credit))}</p>} if image.credit
+      lines << '</figcaption>' if figcaption?(image)
       lines << '</figure>'
       lines.join
     end
 
+    def figcaption?(image)
+      image.caption.present? || image.credit.present?
```

Comment:
Any reason why we care about present? here but not so for `lines << %{<p>#{encode(image.caption)}</p>} if image.caption` ?

Reply:
I normally use `present?` because I've been bitten a few times when evaluating more complex objects.  I'll make sure the lines are consistent.